docs/TRANSLATIONS.md

```diff
@@ -1,116 +1,47 @@
+Translations are maintained using Weblate:
 
-Please respond in the relevant [GitHub discussion](https://github.com/orgs/organicmaps/discussions/8538), or let us know at hello@organicmaps.app
+https://hosted.weblate.org/projects/organicmaps/
 
-## Contribute translations directly
+There is no need to submit translations directly to this repository. Please use Weblate for all translation contributions. Weblate automatically raises pull requests, which are merged by [@organicmaps/mergers](https://github.com/orgs/organicmaps/teams/mergers/members) on a periodic basis.
 
-Adding and updating translations is easy!
-1. Change the translation file you want, e.g. [strings.txt](../data/strings/strings.txt) ([raw text version](https://raw.githubusercontent.com/organicmaps/organicmaps/master/data/strings/strings.txt))
-2. Commit your string changes with the title `[strings] {description of changes}`
-3. (Optional) run the `tools/unix/generate_localizations.sh` script
-4. (Optional) Commit the updated files with the title `[strings] Regenerated`
-5. Send a pull request!
+Weblate components share the same translation memory and vocabulary. The system is configured to automatically propagate translations across components (e.g., from Android to iOS and vice versa). Updating a string in one place should be sufficient.
 
-Please make sure to add a [Developers Certificate of Origin](CONTRIBUTING.md#legal-requirements) to your commit descriptions.
+Please pay close attention to any issues flagged by the automated checks, such as missing string placeholders, inconsistent translations, and other potential errors.
 
-## Requirements
+Components:
 
-To run the `tools/unix/generate_localizations.sh` script, it is necessary to have installed `ruby`.
+- [**Android**](https://hosted.weblate.org/projects/organicmaps/android/) – UI strings for the Android app.
+- [**iOS**](https://hosted.weblate.org/projects/organicmaps/ios/) – UI strings for the iOS app (excluding plurals).
+- [**iOS Plurals**](https://hosted.weblate.org/projects/organicmaps/wip-ios-plurals-please-dont-use/) – UI strings for the iOS app (plurals).
+- [**Sound**](https://hosted.weblate.org/projects/organicmaps/sound/) – Text-to-speech strings for voice navigation.
+- [**Countries**](https://hosted.weblate.org/projects/organicmaps/sound/) – Country and region names.
 
-## Translation files
+## For Developers
 
-- Main:
-  - Application UI strings: [`data/strings/strings.txt`](../data/strings/strings.txt)
-  - A few iOS specific strings: [`iphone/plist.txt`](../iphone/plist.txt)
+Translations are managed by the translation team and do not require direct involvement from developers. Developers should add English base strings using the native tools provided by Android Studio and Xcode.
 
-- POI Categories:
-  - Names of map features/types: [`data/strings/types_strings.txt`](../data/strings/types_strings.txt)
-  - Search keywords/aliases/synonyms for map features: [`data/categories.txt`](../data/categories.txt)
```

Comment:
We should retain the docs for non-Weblate translation files.

Reply:
Retained.